### DISCUSSION TITLE
카카오 사태

### DISCUSSION THREAD

**Discussion by cyberjam** [LifeIsAlone/public-home#65] [General]
청약이 끝난 뒤라 유입이 많진 않지만 다행히 빗겨갔네요. 카카오맵 사용한 다른 프로젝트는 아직 먹통이더라구요.

**Comment by sangjuun-park** [comment]
카카오톡 지금은 잘되지 않나요? ㅋㅋ 아직도 먹통이라니 신기하네요

**Reply by cyberjam** [reply]
넵 ㅎㅎ 카카오톡도 이미지는 안되긴 하지만요. 아직 카카오맵은 안되고 있더라구요 ㅠ
